modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -125,8 +128,8 @@ ComputeMultiPlasticityStress::computeQpStress()
   if (_fspb_debug == 3)
```

Comment:
yea, enum is better, thanks - that's quite old code that i hadn't looked at in months